config/apply/ccsm/translate.js

```diff
@@ -435,53 +424,47 @@ export function translateResponse(customApiResponse, patientData) {
       conclusionCodes = mapResults(excisionResults, customExcisionCodes, standardHistologyCodes, conclusionCodes);
     }
 
-    if (diagnosticReportIndex !== -1) {
-      console.log('Found the diagnostic report reference');
-      // Update the DiagnosticReport in patientData with the results from the custom API
-      patientData[diagnosticReportIndex] = {
-        ...patientData[diagnosticReportIndex],
-        code: {
-          coding: [
-            ...patientData[diagnosticReportIndex].code.coding,
-            ...codings
-          ]
-        },
-        conclusionCode: conclusionCodes
-      };
+    let patient = patientData.find(pd => pd.resourceType === 'Patient');
+
+    // Create a DiagnosticReport resource from Order
+    let newDiagnosticReport = {
+      'resourceType': 'DiagnosticReport',
+      'id': order.OrderId,
```

Comment:
We already have this as a variable from destructuring `order` at the top of the `translateResponse()` function
```suggestion
      'id': orderId,
```

Reply:
Resolved in commit https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/27/commits/031052fe39f46a93d1b94228482d8e09b0605340